torch_xla/csrc/tensor.cpp

```diff
@@ -1273,8 +1280,9 @@ XLATensor::ComputationCache* XLATensor::GetComputationCache() {
 }
 
 XLATensor::PostOrderData XLATensor::RunPostOrder(
-    const std::vector<XLATensor>& tensors, absl::Span<const size_t> indices) {
-  std::vector<const torch::lazy::Node*> roots;
+    const std::vector<XLATensor>& tensors, SyncTensorCollection* coll) {
+  absl::Span<const size_t> indices = coll->indices;
+  std::vector<const ir::Node*> roots;
```

Comment:
This should be `std::vector<const torch::lazy::Node*> roots;`. I think this might've been the cause of the build error. Can you try updating it and build locally?

Reply:
Good catch @wonjoolee95. Thanks!